api/apirouter.go

```diff
@@ -35,6 +35,8 @@ func NewAPIRouter(app *appContext, userRealIP bool) apiMux {
 
 	mux.HandleFunc("/status", app.status)
 
+	mux.Get("/ticket_price", app.TicketPrices)
```

Comment:
There is a `/stake` router with a `/diff` subrouter. Within that router, there is already the following endpoint:

`Get("/r/{idx0}/{idx}", app.getStakeDiffRange)`

What differentiates this new endpoint from that one?  Move and rename the new endpoint accordingly.  (It shows the ticket price intervals).